StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsTheme.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  FinancialConnectionsTheme.swift
+//  StripeFinancialConnections
+//
+//  Created by Mat Schmid on 2024-07-17.
+//
+
+import UIKit
+
+enum FinancialConnectionsTheme: String, SafeEnumCodable, Equatable {
+    case light = "light"
+    case dashboardLight = "dashboard_light"
+    case linkLight = "link_light"
+    case unparsable
+}
+
+extension FinancialConnectionsTheme? {
+    var logo: Image {
+        switch self {
+        case .linkLight:
+            return .link_logo
+        case .light, .dashboardLight, .unparsable, .none:
+            return .stripe_logo
+        }
+    }
+
+    var primaryColor: UIColor {
+        switch self {
+        case .linkLight:
+            return .linkGreen200
+        case .light, .dashboardLight, .unparsable, .none:
+            return .brand500
+        }
+    }
```

Comment:
I like how this turned out in the sense that colors are still "theme independent" (doesn't know about theme)

And instead a Theme wraps colors

Makes sense; at the start I was (naively) afraid whether theme would leak into all the lower-level API's (ex. color or fonts) but that's not necessary